src/auxiliary/t8code.jl

```diff
@@ -0,0 +1,465 @@
+"""
+    init_t8code()
+"""
+function init_t8code()
+  t8code_package_id = t8_get_package_id()
+  if t8code_package_id >= 0
+    return nothing
+  end
+
+  # Initialize `p4est` with log level ERROR to prevent a lot of output in AMR simulations
```

Comment:
```suggestion
  # Initialize `t8code` with log level ERROR to prevent a lot of output in AMR simulations
```

Reply:
Done.

Reply:
You can resolve comments like this when you do the changes. This makes it easier for reviewers to concentrate on the rest